tests/test_workers/test_api_utils.py

```diff
@@ -76,6 +76,7 @@ def test_update_request_connection_failed(mock_session):
 
     with pytest.raises(IIBError, match='The connection failed.+'):
         api_utils.update_request(3, {'index_image': 'index-image:latest'})
+        assert mock_session.patch.call_count == 5
```

Comment:
Is it possible to use config here?

Reply:
Ah, that's a nice suggestion. Changed this one :)